x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/action/user/TransportSetEnabledActionTests.java

```diff
@@ -252,14 +259,87 @@ public void onFailure(Exception e) {
         );
     }
 
+    public void testUserCanModifySameNameUserFromDifferentRealm() throws Exception {
```

Comment:
This method is very similar to the existing `testValidUser(User)`. Can we somehow consolidate them?

Reply:
Yup, should be able to pass in `authentication` - rest is generic